build_n_deploy/webpack/webpack.common.js

```diff
@@ -57,9 +48,14 @@ const common = {
             inject: 'body',
             alwaysWriteToDisk: true,
         }),
-        new TypeScriptTypeChecker(),
-        new ESLintPlugin({
-            extensions: [`ts`, `tsx`],
+        new ForkTsCheckerWebpackPlugin({
```

Comment:
For at denne skal fungere med babel må vi sette typescript-propsene. Satte async:true, men det skal i teorien bli async i dev - just in case.

Reply:
👀